pkgs/development/beam-modules/rebar3-nix/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, buildRebar3, fetchFromGitHub }:
+let
+  name = "rebar3_nix";
+  version = "0.1.0";
+in buildRebar3 {
+  inherit name version;
```

feel like this is a little more readable, but don't feel strongly
```suggestion
buildRebar3 rec {
  name = "rebar3_nix";
  version = "0.1.0";
```